pootle/apps/pootle_statistics/views.py

```diff
@@ -49,40 +73,26 @@ def get_context_data(self, *args, **kwargs):
         )
 
 
-class TPContributorsJSON(ContributorsJSONMixin, TPBrowseView):
-    pass
-
-
-class TPStoreContributorsJSON(ContributorsJSONMixin, TPBrowseStoreView):
-    pass
-
-
-class LanguageContributorsJSON(ContributorsJSONMixin, LanguageBrowseView):
-    pass
-
+def create_stats_view_class(mixin_class, base_class):
```

Comment:
im not really understanding the point of this, and i feel as tho it increases complexity/implicitness